<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationClass</class>
 <widget class="QMainWindow" name="TransformationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Polygón</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="draw">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Vykreslit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Vymazat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="color">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Farba</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>211</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Transformácie</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="rotateRight">
       <property name="text">
        <string>Rotácia vpravo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotateLeft">
       <property name="text">
        <string>Rotácia vľavo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomIn">
       <property name="text">
        <string>Zväčšiť</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOut">
       <property name="text">
        <string>Zmenšiť</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flip">
       <property name="text">
        <string>Preklopiť</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shearplus">
       <property name="text">
        <string>Skosiť +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shearminus">
       <property name="text">
        <string>Skosiť -</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>641</width>
      <height>561</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>639</width>
       <height>559</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radio1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DDA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>620</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bresenham</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Transformation.qrc"/>
 </resources>
 <connections/>
</ui>
